Buildings/Air/Systems/SingleZone/VAV/BaseClasses/Validation/HysteresisWithHold.mo

```diff
@@ -0,0 +1,33 @@
+within Buildings.Air.Systems.SingleZone.VAV.BaseClasses.Validation;
+model HysteresisWithHold
+  extends Modelica.Icons.Example;
+  Modelica.Blocks.Sources.Sine pulse1(amplitude=0.2, freqHz=1/360)
+    annotation (Placement(transformation(extent={{-40,-10},{-20,10}})));
+  Buildings.Air.Systems.SingleZone.VAV.BaseClasses.HysteresisWithHold onHold_30sec(offHolDur=
```

Comment:
Please use one parameter assignment per line for easier readability, and add comments to all instances